t/rebase.t

```diff
@@ -15,6 +15,7 @@ use Test::Warn;
 use lib 't/lib';
 use MockOutput;
 use TestConfig;
+use Carp::Always;
```

Comment:
I don't think you need this if you're using `hurl` and as long as the tests test for warnings.

Reply:
Yes, agreed, this snuck in somehow.